modules/dnn/src/layers/detection_output_layer.cpp

```diff
@@ -190,7 +196,7 @@ class DetectionOutputLayerImpl CV_FINAL : public DetectionOutputLayer
         _locPredTransposed = getParameter<bool>(params, "loc_pred_transposed", 0, false, false);
         _bboxesNormalized = getParameter<bool>(params, "normalized_bbox", 0, false, true);
         _clip = getParameter<bool>(params, "clip", 0, false, false);
-        _groupByClasses = getParameter<bool>(params, "group_by_classes", 0, false, true);
+        _groupByClasses = getParameter<bool>(params, "group_by_classes", 0, false, false);
```

Comment:
@dkurt Could you please take a look on this line more carefully?
This change will diverge code agains 3.4 branch.

Reply:
@YashasSamaga checked that this flag is not used in our models. So We can freely switch it to `false` by default.